RecoJets/JetProducers/src/PileupJetIdAlgo.cc

```diff
@@ -338,6 +338,7 @@ PileupJetIdentifier PileupJetIdAlgo::computeIdVariables(const reco::Jet* jet,
   for (unsigned i = 0; i < jet->numberOfSourceCandidatePtrs(); ++i) {
     reco::CandidatePtr pfJetConstituent = jet->sourceCandidatePtr(i);
     const reco::Candidate* icand = pfJetConstituent.get();
+    assert(icand);
```

Comment:
just to silence the clang-analyzer warning.  clang analyzer clains that ` pfJetConstituent.get()` can return `nullptr`